packages/vite/src/clientSsr.ts

```diff
@@ -153,7 +156,10 @@ export function renderFromDist<TProps extends Record<string, any>>(
     const stream = renderToReadableStream(
       // createElement(layout, undefined, createElement(page, props)),
       // @ts-expect-error - WIP
-      createElement(ServerEntry, { location: { pathname: rscId } }),
+      createElement(ServerEntry, {
+        location: { pathname: rscId },
+        css: cssLinks,
+      }),
```

Comment:
Really nice implementation 🥇

Reply:
Thank! The way that Tobbe and Danny laid the groundwork here made it somewhat easy to pass the collected css links through and have them loaded.